services/api-server/src/simcore_service_api_server/api/routes/_common.py

```diff
@@ -1,8 +1,10 @@
-from typing import Any
+from typing import Any, Final
 
 from fastapi import status
 
-JOB_OUTPUT_LOGFILE_RESPONSES: dict[int, dict[str, Any]] = {
+from ...core.settings import BasicSettings
+
+job_output_logfile_responses: dict[int | str, dict[str, Any]] = {
```

Comment:
why the small casing here?
if it's because of Ruff. I think I removed that rule cause I thought that goes against our conventions?

Reply:
yes, it was upper-case and I guess I lowercased it because of ruffus :-)